### PR TITLE
add beta command

### PR DESCRIPTION
**Pull request by zc2638**
Signed-off-by: zc <zc2638@qq.com>

**What type of PR is this?**

/kind feature

**What this PR does / why we need it**:

Added `beta` commands to unify beta-type command operations.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by zc2638** [comment]
#3540 @zhu733756 
#3537 @gy95

**Comment by zhu733756** [comment]
Seems good.

File: keadm/cmd/keadm/app/cmd/beta/beta.go [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+    You can then repeat the second step on, as many other machines as you like.
+`
+)
+
+// NewBeta represents the beta command
+func NewBeta() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "beta",
+		Short:   "keadm beta version operation",
```
Comment (fisherxu) [inline keadm/cmd/keadm/app/cmd/beta/beta.go:60]:
```suggestion
		Short:   "keadm beta version sub-commands",
```
?

File: keadm/cmd/keadm/app/cmd/beta/beta.go [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    | master node (on the cloud)# sudo keadm beta init         |
+    +----------------------------------------------------------+
+
+    +----------------------------------------------------------+
+    | On the edge machine:                                     |
+    +----------------------------------------------------------+
+    | worker node (at the edge)# sudo keadm beta join <flags>  |
+    +----------------------------------------------------------+
+
+    You can then repeat the second step on, as many other machines as you like.
```
Comment (fisherxu) [inline keadm/cmd/keadm/app/cmd/beta/beta.go:52]: We have the introdunction of keadm when only running `keadm`, so this description should descripts the meaning of the `beta` sub-command itself, only a short maybe enough.
We can add more description in `beta init` and `beta join` and so on. How do you think? @zhu733756 @zc2638 @gy95

**Review by fisherxu — COMMENTED** [review]
Looks good to add beta sub-cmd :)

File: keadm/cmd/keadm/app/cmd/beta/beta.go (hunk 2 above)
Comment (zc2638) [reply keadm/cmd/keadm/app/cmd/beta/beta.go:52]: I agree with you.
At first thought, the beta version of keadm does, so bring the full description in. 
When the subcommands under beta are migrated to the main command, beta can still be used, but it is not appropriate to describe the overall keadm.
@fisherxu

**Review by fisherxu — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**